addons/mail/static/src/messaging/entity/entity_core.js

```diff
@@ -65,6 +65,19 @@ function _checkFields(Entities) {
                     }" has no inverse field "${RelatedEntity.name}/${field.inverse}".`
                 );
             }
+            if (inverseField.inverse !== fieldName) {
+                throw new Error(
+                    `Relation "${
+                        Entity.name
+                    }/${
+                        fieldName
+                    }" field name do not match with inverse field inverse name definition "${
```

Comment:
_Relation $1/$2 field name do not match with inverse field inverse name definition $3/$4._
=>
_Inverse field name of relation $1/$2 does not match with field name of relation $1/$2._

I think the sentence is clearer that way, and gives more clue to how to fix it.

Reply:
Ahah thanks, I didn't find a way to be clear :p